ts/src/pro/coinex.ts

```diff
@@ -141,10 +141,21 @@ export default class coinex extends coinexRest {
             const symbol = this.safeSymbol (marketId, undefined, undefined, defaultType);
             const market = this.safeMarket (marketId, undefined, undefined, defaultType);
             const parsedTicker = this.parseWSTicker (rawTicker, market);
-            const messageHash = 'ticker:' + symbol;
             this.tickers[symbol] = parsedTicker;
             newTickers.push (parsedTicker);
-            client.resolve (parsedTicker, messageHash);
+        }
+        const messageHashes = this.findMessageHashes (client, 'tickers:');
+        for (let i = 0; i < messageHashes.length; i++) {
+            const messageHash = messageHashes[i];
+            const parts = messageHash.split (':');
```

Comment:
@pcriadoperez hum we can't use `:` here safely, what if you have `messageHas = 'tickers:BTC/USDT,BTC/USDT:USDT'` 

Then parts would be `['tickers', 'BTC/USDT,BTC/USDT', 'USDT']`

Reply:
good catch.. I replaced it for '::'